cmd/chirpstack-gateway-bridge/cmd/configfile.go

```diff
@@ -88,6 +88,9 @@ type="{{ .Backend.Type }}"
   # the time would otherwise be unset.
   fake_rx_time={{ .Backend.SemtechUDP.FakeRxTime }}
 
+  # Cleanup duration 
```

Comment:
Could we rename this to `connection_timeout_duration`? I think the description should be something like:

```
# Connection timeout duration
#
# ChirpStack Gateway Bridge keeps a list of connected gateways. If a gateway does not send any
# UDP data within the configured timeout, then ChirpStack Gateway Bridge will consider the gateway
# disconnected and it will unsubscribe from the gateway MQTT topic and cleanup the UDP docket.
```

This duration has nothing to do with Class-C devices, although it might help to work around the Class-C issue you are experiencing.